src/internal/base-component/use-base-component.ts

```diff
@@ -9,8 +9,10 @@ import {
   useComponentMetadata,
 } from "@cloudscape-design/component-toolkit/internal";
 
-import { PACKAGE_SOURCE, PACKAGE_VERSION } from "../environment";
+import { PACKAGE_SOURCE, PACKAGE_VERSION, THEME } from "../environment";
+import { getVisualTheme } from "../utils/get-visual-theme";
```

Comment:
one meta-question: why do we have this utils copied to all packages instead of `component-toolkit`?

Reply:
Coming in the future: https://github.com/cloudscape-design/component-toolkit/pull/122